### DISCUSSION TITLE
Mirage not working

### DISCUSSION THREAD

**Discussion by p4ranoid4** [qwbarch/mirage#105] [Q&A]
I need help, I played with the skinwalker mod for 2 months I have over 100 mods and the mod still worked all the time then I posted below a video on youtube and there someone wrote that I should immediately download mirage it's better everything and that I can customize it how I want but neither thing works with it, I figured it might be that the mods are incompatible so I created a new slot and downloaded just mirage and lethalconfig, I played it with a friend for about 20 minutes and neither mob talked and I couldn't edit it in either lethalsetting or lethal config, I'm still looking for a way to fix the mirage mod, I don't know what to do, I'll try again today and see if im gonna  download skinwalker again
![obrázek_2024-11-09_165623262](https://github.com/user-attachments/assets/cb1403ac-20d4-487a-bad9-7fad8b14c3fd)

**Comment by p4ranoid4** [comment]
otherwise we don't speak English in skinwalker it wasn't a problem it's hard to say if it is here

**Comment by p4ranoid4** [comment]
i found the option where i can edit the mod.

**Comment by qwbarch** [comment]
Sorry for the late reply, I've been busy and haven't been looking at GitHub.
Based on your last comment, is your problem no longer in issue?

**Reply by p4ranoid4** [reply]
everything is fine mimic talked but I would still like to ask if I can set it to talk all the mobs if it is possible if the mod is only intended for mimic so do not mind but if it can somehow be set so I'm wondering how and I'm also wondering that if I have a lot of mods and the mod doesn't work, it means that some mod doesn't work with that mod together and I'd really like to know if it's possible to find out what mod
thanks for reply!

**Reply by qwbarch** [reply]
You can enable/disable any monster to mimic voices by editing ``Mirage.Enemies.cfg``.

> I'm also wondering that if I have a lot of mods and the mod doesn't work, it means that some mod doesn't work with that mod together

Mod conflicts *can* be a reason for a mod not working, but it's not always the case and could also be the mod itself. In any case, I'd always recommend asking about the issue if you're not sure (as you already have).

If you have any questions feel free to ask!
